packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -356,116 +357,127 @@ const removeNode = (event) => {
 	workflowService.removeNode(wf.value, event);
 };
 
-const largeNode = { width: 420, height: 220 };
+const addOperatorToWorkflow: Function = (operator: any) => () => {
+	workflowService.addNode(wf.value, operator.operation, newNodePosition);
+	workflowDirty = true;
+};
+const addLargeOperatorToWorkflow: Function = (operator: any) => () => {
+	workflowService.addNode(wf.value, operator.operation, newNodePosition, {
+		size: { width: 420, height: 220 }
+	});
+	workflowDirty = true;
+};
 
 // Menu categories and list items are in order of appearance for separators to work
-const categories: Record<string, { label: string; icon: string; separator?: boolean }> = {
-	model: {
+const contextMenuItems: MenuItem[] = [
+	// Model
+	{
 		label: 'Model operators',
-		icon: 'pi pi-share-alt'
+		items: [
+			{
+				label: ModelOp.operation.displayName,
+				command: addOperatorToWorkflow(ModelOp)
+			},
```

Comment:
This was previously generated, so we didn't have to repeat label/command over and over to keep the 3rd-party interface at a minimum. What are the benefits of changing this?

Reply:
The use of function to avoid a repetition when we have to list all the `name` none the less mase the code unclear. Now we also have to group those operator. So listing them is non-negotiable.
This is not DRY, but it's clear, and simple to understand what is happening.